ampersand-select-view.js

```diff
@@ -189,6 +196,21 @@ SelectView.prototype.clear = function() {
     return this;
 };
 
+/**
+ * Sets control to option with the same value as initial value if specified, falls 
+ * back to either unselectedText or first option depending on availability
+ * @return {SelectView} this
+ */
+SelectView.prototype.reset = function() {
+    if(this.startingValue) {
+        this.setValue(this.startingValue, true);
+    } else {
```

Comment:
`this.select.length > 0` should be checked here instead, to prevent among other things a value of `0` in `options[0]` from being ignored.